### DISCUSSION TITLE
Env File Error

### DISCUSSION THREAD

**Discussion by Tanvir-rahman** [saadeghi/daisyui#2577] [Q&A]
I am trying to run daisyui locally. After running `npm run init` getting env file related error.
`Error executing command: node: bad option: --env-file=.env.local`

What will be the .env.local file's content?

**Comment by saadeghi** [comment]
Create an empty `.env.local` file at `src/docs` and it should work


I will add this in the contribution guide.
Let me know if you have any questions
